xbmc/music/karaoke/GUIDialogKaraokeSongSelector.h

```diff
@@ -83,5 +83,5 @@ class CGUIDialogKaraokeSongSelectorLarge : public CGUIDialogKaraokeSongSelector
 {
   public:
     CGUIDialogKaraokeSongSelectorLarge();
-    void Open();
+    void Open(const std::string &param = "");
```

Comment:
To avoid this kind of confusion or human error please start using the `override` keyword to mark methods that are meant to override a virtual method of a base class. The compiler will then throw an error if it doesn't actually override an existing virtual method (e.g. because the base method is not virtual, or the parameter list doesn't match).